src/Microsoft.ML.TensorFlow/TensorFlow/Tensorflow.cs

```diff
@@ -24,9 +22,9 @@
 using TF_DeviceList = System.IntPtr;
 
 using size_t = System.UIntPtr;
-using System.Numerics;
 using System.Collections.Generic;
-using System.Linq.Expressions;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Data;
```

Comment:
Are these using statements actually necessary?  I'm missing the additions that actually used them.

Reply:
Thanks for pointing it out. I added the new API in this file, but ended up moving it to TensorflowUtils.

---
In reply to: [216493797](https://github.com/dotnet/machinelearning/pull/862#discussion_r216493797) [](ancestors = 216493797)